stories/15.SequenceHeatmap/NightingaleSequenceHeatmap.stories.ts

```diff
@@ -0,0 +1,799 @@
+import { Meta, Story } from "@storybook/web-components";
+import { html } from "lit-html";
+import "../../packages/nightingale-sequence-heatmap/src/index.ts";
+
+export default {
+  title: "Components/Other/SequenceHeatmap",
+} as Meta;
+
+const Template: Story<{
+  elementId: string;
+  width: number;
+  height: number;
+  "display-start": number;
+  "display-end": number;
+  "highlight-color": string;
+}> = (args) => {
+  const { elementId, width, height } = args;
+  return html`<nightingale-sequence-heatmap
+    heatmapId=${elementId}
+    width="${width}"
+    height=${height}
+    display-start=${args["display-start"]}
+    display-end=${args["display-end"]}
+    highlight-event="onmouseover"
+    highlight-color=${args["highlight-color"]}
+  ></nightingale-sequence-heatmap>`;
+};
+
+export const SequenceHeatmap = Template.bind({});
+SequenceHeatmap.args = {
+  width: 500,
+  height: 500,
+  elementId: "heatmapId1",
+};
+SequenceHeatmap.play = async () => {
+  await customElements.whenDefined("nightingale-sequence-heatmap");
+  const heatmapElement = document.getElementsByTagName(
+    "nightingale-sequence-heatmap",
+  )[0] as any;
+  const xDomain = [
```

Comment:
Maybe import these values from your mockup data to simplify the code of the story.

Reply:
Fixed